R-package/src/lightgbm_R.h

```diff
@@ -465,13 +468,16 @@ LIGHTGBM_C_EXPORT SEXP LGBM_BoosterPredictForFile_R(
 
 /*!
 * \brief Get number of prediction
-* \param handle handle
-* \param num_row
-* \param is_rawscore
-* \param is_leafidx
+* \param handle Booster handle
+* \param num_row number of rows in input
+* \param is_rawscore 1 to get raw predictions, before transformations like
+*                    converting to probabilities, 0 otherwise
+* \param is_leafidx 1 to get record of which leaf in each tree
+*                   observations fell into, 0 otherwise
+* \param is_predcontrib 1 to get feature contributions, 0 otherwise
```

Comment:
Missed `start_iteration`.

Reply:
updated in https://github.com/microsoft/LightGBM/pull/4257/commits/9c173115582f1bb3793456beabbc1ace2b778899